core/commands/swarm.go

```diff
@@ -369,7 +370,8 @@ var swarmDisconnectCmd = &cmds.Command{
 'ipfs swarm disconnect' closes a connection to a peer address. The address
 format is an IPFS multiaddr:
 
-ipfs swarm disconnect /ip4/104.131.131.82/tcp/4001/ipfs/QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ
+ipfs swarm disconnect /ip4/104.131.131.82/tcp/4001/ipfs/QmaCpDMGvV2BGHeYERUEnR
+QAwe3N8SzbUtfsmvsqQLuvuJ
```

Comment:
Do not split a hash like this.  put it on its on line, if it still too long then I would leave it.